src/staticmaps.js

```diff
@@ -414,6 +414,18 @@ class StaticMaps {
    */
   getTile(data) {
     return new Promise((resolve, reject) => {
+      // custom for rendering map without base layer
+      if( data.url === null ){
```

Comment:
Can you check the lint error? I think it's fixed by doing this:
```
      if (data.url === null) {
        resolve({
          success: true,
          tile: {
            url: data.url,
            box: data.box,
            body: null,
          },
        });
        return;
      }
```

Reply:
done :)